python/tank_vendor/shotgun_authentication/ui/ui_login_dialog.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'login_dialog.ui'
+#
+#      by: pyside-uic 0.2.15 running on PySide 1.2.2
+#
+# WARNING! All changes made in this file will be lost!
+
+from .qt_abstraction import QtCore, QtGui
```

Comment:
Minor comment, but our normal convention is that the auto-generated py file has the same name as the ui file - so in this case it would be `ui/login_dialog.py`.